hw03-testframework/src/main/java/ru/otus/testing/TestResult.java

```diff
@@ -0,0 +1,16 @@
+package ru.otus.testing;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public class TestResult {
+    private String testName;
+    private long runningTime;
```

Comment:
Не понятно в каких единицах время.
Секунды, миллисекунды?
В таких случаях лучше единицу измерения записывать в имя поля.
```suggestion
    private long runningTimeInMillis;
```